src/bindings/Vscode.mli

```diff
@@ -306,3 +313,74 @@ module LanguageClient : sig
 
   val initializeResult : t -> InitializeResult.t Promise.t
 end
+
+module ShellExecution : sig
+  type shellQuotingOptions
+
+  type shellExecutionOptions =
+    { cwd : string option
+    ; env : string Js.Dict.t option
+    ; executable : string option
+    ; shellArgs : string list option
+    ; shellQuoting : shellQuotingOptions option
+    }
+
+  type t =
+    { args : string array option
+    ; command : string option
+    ; commandLine : string option
+    ; options : shellExecutionOptions option
+    }
+
+  val make : commandLine:string -> options:shellExecutionOptions option -> t
+end
+
+module ProcessExecution : sig
+  type processExecutionOptions =
+    { cwd : string option
+    ; env : string Js.Dict.t option
+    }
+
+  type t =
+    { args : string array
+    ; options : processExecutionOptions option
+    ; process : string
+    }
+
+  val make :
+       process:string
+    -> args:string array
+    -> options:processExecutionOptions option
+    -> t
+end
+
+module Task : sig
+  type t
+
+  type taskDefinition = { type_ : string [@bs.as "type"] }
+
+  val make :
+       taskDefinition:taskDefinition
+    -> scope:[ `Folder of Folder.t ]
+    -> name:string
+    -> source:string
+    -> execution:[ `Shell of ShellExecution.t | `Process of ProcessExecution.t ]
+    -> problemMatchers:string list option
```

Comment:
Is there a difference between `Some []` and `None` here?

Reply:
I think it should be the same, will remove the option